cg/meta/orders/microbial_fastq_submitter.py

```diff
@@ -0,0 +1,117 @@
+from datetime import datetime
+
+from cg.constants import Workflow, DataDelivery
+from cg.exc import OrderError
+from cg.meta.orders.lims import process_lims
+from cg.meta.orders.submitter import Submitter
+from cg.models.orders.order import OrderIn
+from cg.models.orders.sample_base import StatusEnum
+from cg.store.exc import EntryNotFoundError
+from cg.store.models import Sample, Customer, Case, CaseSample
+from cg.utils.error_handler import handle_errors
+
+
+class MicrobialFastqSubmitter(Submitter):
```

Comment:
I'm wondering if we are missing logic for existing samples? Usually these are identified by already having an internal_id filled in. Such samples should not be added to StatusDB and to my knowledge we would only need to connect them to the generated case.